### PR TITLE
Fix travis tests   backend

### PR DESCRIPTION
**Pull request by GHNelson752**

### CODE REVIEW

File: model/dbCommands.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
 const mysql = require('mysql');
 const util = require('util');
-const dbconfig = require('../dbconfig.json');
+// const dbconfig = require('../dbconfig.json');
```
Comment (atracey1) [inline model/dbCommands.js]: This comment can probably go now

File: model/dbCommands.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+const dbconfig = {
+  host: process.env.DB_HOST,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASS,
+  connectionLimit: process.env.DB_CON_LIMIT,
+  database: process.env.DB_DATABASE,
+};
 
 function wrapDB(dbConfig) {
+  console.log(dbconfig);
```
Comment (atracey1) [inline model/dbCommands.js]: This log can probably also go

**Review by atracey1 — APPROVED** [review]
LGTM

**Review by nmclarnon — APPROVED** [review]

File: model/dbCommands.js (hunk 1 above)
Comment (GHNelson752) [reply model/dbCommands.js]: Resolved in final clean up commit

File: model/dbCommands.js (hunk 2 above)
Comment (GHNelson752) [reply model/dbCommands.js]: Resolved in final clean up commit

**Pull request merged**